packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.Props.ts

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { OverflowSet } from './OverflowSet';
+import { IContextualMenuItem } from '../../ContextualMenu';
+import { IButtonProps } from '../../Button';
+import { IRenderFunction } from '../../Utilities';
+
+export interface IOverflowSetProps extends React.Props<OverflowSet> {
+
+  /**
+   * An array of items to be rendered by your onRenderItem function in the primary content area
+  */
+  items?: any[];
+
+  /**
+   * An array of items to be passed to overflow contextual menu
+  */
+  overflowItems?: IContextualMenuItem[];
+
+  /**
+   * Method to call when trying to render an item.
+  */
+  onRenderItem: IRenderFunction<any>;
+
+  /**
+   * Rendering method for overflow button and contextual menu.
+  */
+  onRenderOverflowButton: IRenderFunction<IButtonProps>;
```

Comment:
Ah I see. I guess I was expecting the API to list "top level controls" and "overflow controls" in a similar manner. But given the existing mechanism for overriding menu items in contextualmenuitem, this seems to make sense and also allows customization of the button itself.